gst/nnstreamer/nnstreamer_log.h

```diff
@@ -56,13 +61,17 @@
 #define ml_logd(...) \
     __android_log_print (ANDROID_LOG_DEBUG, TAG_NAME, __VA_ARGS__)
 
+#define ml_logc(...) \
+    __android_log_print (ANDROID_LOG_FATAL, TAG_NAME, __VA_ARGS__)
+
 #else /* Linux distro */
 #include <glib.h>
 
 #define ml_logi g_info
 #define ml_logw g_warning
-#define ml_loge g_critical
+#define ml_loge g_error
```

Comment:
Q: g_error always fatal. is it ok?

Reply:
It is neither Tizen or Android, so it isn't that critical, but yes, it is ok as long as we intend to use g_error as a standard message for "error" log. (GLIB is the standard in NNStreamer)

Reply:
Ah.. or is "g_error" more "critical" than "g_critical"?

Reply:
OK. error is "fatal" and critical is "critical-warning". I'll rename it ml_logf (fatal).